sdk/signalr/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/rest.py

```diff
@@ -0,0 +1,942 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+__all__ = [
+    'build_add_connection_to_group_request',
+    'build_add_user_to_group_request',
+    'build_connection_exists_request',
+    'build_group_exists_request',
+    'build_check_permission_request',
+    'build_user_exists_request',
+    'build_close_client_connection_request',
+    'build_grant_permission_request',
+    'build_healthapi_get_health_status_request',
+    'build_remove_connection_from_group_request',
+    'build_remove_user_from_all_groups_request',
+    'build_remove_user_from_group_request',
+    'build_revoke_permission_request',
+    'build_send_to_all_request',
+    'build_send_to_connection_request',
+    'build_send_to_group_request',
+    'build_send_to_user_request'
```

Comment:
I'm guessing there are no operations groups for this API?
I saw there's been a shift to include operation groups in the namespace rather than the function name - but it seems that change has no impact on this SDK?

Reply:
@annatisch @johanste has already done some manual-manipulation of the code, we completely got rid of the operation groups